anesthetic/gui/plot.py

```diff
@@ -245,8 +252,8 @@ def points(self, label):
     def update(self, _):
         """Update all the plots upon slider changes."""
         logX = np.log(self.samples.nlive / (self.samples.nlive+1)).cumsum()
-        kT = self.temperature()
-        LX = self.samples.logL/kT + logX
+        beta = self.temperature()
+        LX = self.samples.logL*beta + logX
```

Comment:
This relates to my comment above, `beta = temperature` is a bit confusing, so I think we should rename the temperature slider to a beta slider.

Reply:
done